zerver/decorator.py

```diff
@@ -236,8 +236,12 @@ def access_user_by_api_key(request, api_key):
     if user_profile.realm.deactivated:
         raise JsonableError(_("Realm for account has been deactivated"))
 
-    if not check_subdomain(get_subdomain(request), user_profile.realm.subdomain):
-        logging.warning("User %s attempted to access webhook API on wrong subdomain %s" % (
+    if (not check_subdomain(get_subdomain(request), user_profile.realm.subdomain) and
+        # Allow access to localhost for Tornado
```

Comment:
It's pretty smelly to have a comment inside a conditional.  Does it make sense to either extract a helper function or use nested ifs here?  Or maybe just move the comment.